dds/idl/be_global.cpp

```diff
@@ -748,6 +748,18 @@ ExtensibilityKind BE_GlobalData::extensibility(AST_Decl* node) const
   return value;
 }
 
+TryConstructFailAction BE_GlobalData::try_construct(AST_Decl* node) const
+{
+  TryConstructAnnotation* try_construct_annotation =
+    dynamic_cast<TryConstructAnnotation*>(
+      builtin_annotations_["::@try_construct"]);
+  TryConstructFailAction value;
+  if (!try_construct_annotation->node_value_exists(node, value)) {
+    value = tryconstructfailaction_discard;
+  }
+  return value;
```

Comment:
```suggestion
  return try_construct_annotation->node_value(node);
```

If we don't need to have a command line option for this (I'm not sure we do or not), then this can be simplified a little by implementing `TryConstructAnnotation::default_value()` to return `tryconstructfailaction_discard`.